packages/api/src/routes/nfts-list.js

```diff
@@ -1,8 +1,8 @@
-import { Validator } from '@cfworker/json-schema'
+import { HTTPError } from '../errors.js'
 import { JSONResponse } from '../utils/json-response.js'
-import { validate } from '../utils/auth.js'
+import { Validator } from '@cfworker/json-schema'
 import { toNFTResponse } from '../utils/db-transforms.js'
-import { HTTPError } from '../errors.js'
+import { validate } from '../utils/auth.js'
```

Comment:
Please revert all these import order changes in the API. It is more time consuming for reviewers to review when there are no material changes in your diff because reviewers have to look at it to figure out there was nothing added or removed. It also triggers "release please" to create a release PR for the API and include this PR in the changelog.

If you want to run a code formatter against the code then the changes should be sent in a separate PR.